src/libostree/ostree-repo.c

```diff
@@ -2094,6 +2094,27 @@ append_remotes_d (OstreeRepo          *self,
 }
 
 gboolean
+ostree_repo_is_created (OstreeRepo    *self,
+                        gboolean      *does_exist,
+                        GError        **error)
+{
+  struct stat stbuf;
+
+  if (lstat (gs_file_get_path_cached (self->objects_dir), &stbuf) < 0)
```

Comment:
No, I meant only have `ostree_repo_is_created()` as the new API, but inside, instead of stat'ing the objects dir, you would call `ostree_repo_open()`, since it pretty much goes through the whole process of verifying that the repo is created as it opens it. But of course, you don't want to change the `OstreeRepo` object that was passed to you, so you'd have to create a new reference from that one. But again, I don't mind getting something like that in later as well.